ci/purge_expired_questionnaire_state.yaml

```diff
@@ -0,0 +1,44 @@
+platform: linux
+image_resource:
+    type: docker-image
+    source:
+        repository: gcr.io/google.com/cloudsdktool/cloud-sdk
+params:
+    SERVICE_ACCOUNT_JSON: ((gcp.service_account_json))
+    PROJECT_ID:
+    DATAFLOW_TEMPLATE_VERSION: '2020-11-30-00_RC00'
```

Comment:
Is this the latest DATAFLOW_TEMPLATE_VERSION? It might have moved on since since purge_expired_sessions was done